deepeval/models/hallucination_model.py

```diff
@@ -0,0 +1,18 @@
+import os
+from deepeval.singleton import Singleton
+from sentence_transformers import CrossEncoder
+from deepeval.models.model_map import model_map, name_to_card
+
+
+class HallucinationModel(metaclass=Singleton):
+    def __init__(self, model_name: str = "vectara-hallucination"):
+        # We use a smple cross encoder model
+        os.environ["TOKENIZERS_PARALLELISM"] = "false"
+
+        # TODO: add this progress context in the correct place
```

Comment:
quick question, how long does it take the model to load? Is there a reason why this is left commented out?